eclipse-theia/theia
# Update Themes

#### What it does

Updates the specification of our default themes to match recent VSCode.

#### How to test

1. Open a TS file.
2. Move through the built-in themes.
3. Observe that they behave as expected.

#### Review checklist


CQ: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=23884

#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
